docs/DevGuide/Travis.md

```diff
@@ -74,6 +74,29 @@ performed in the CHECK_FILES build.
 using a Linux OS, and the `AppleClang` compiler for `OS X`.
 * The `gcc Debug` build will fail if there are `doxygen` warnings.
 
+## How to perform the checks locally
+
+Before pushing to GitHub and waiting for Travis to perform the checks it is
+useful to perform at least the following tests locally:
+- **Unit tests:** Perform a `make test-executables` and then `ctest -L unit` to
+  run all unit tests. This can take a significant amount of time, so to run only
+  a subset of the tests you can use one of the other keywords that the tests are
+  labeled with, such as `ctest -L datastructures`. To run only a particular test
+  you can also execute `ctest -R TEST_NAME` instead, where `TEST_NAME` is the
+  test identifier such as `Unit.DataStructures.Mesh`. The output of these tests
```

Comment:
`identifier` -> `regular expression`, you should mention that you can also do just `Mesh` to run all the mesh-related tests that have `Mesh` in the name